genre.rb

```diff
@@ -1,17 +1,14 @@
-require './item'
-
 class Genre
-  attr_accessor :name
-  attr_reader :items
+  attr_reader :name, :items
 
-  def initialize(_name)
-    @id = Random.rand(1..10_00)
+  def initialize(name, id = Random.rand(1..1000))
+    @id = id
```

Comment:
- [ ] Would you please elaborate why you need those changes? The previous code working, in my opinion. The mismatching name and _name can be corrected rightfully, though. Like
def initialize(name)
 \@name = name